gradle/libs.versions.toml

```diff
@@ -11,10 +11,10 @@ kotlinxCoroutinesVersion = "1.6.4"
 androidAppCompatVersion = "1.6.1"
 
 # android compose
-composeUiVersion = "1.5.1"
+composeUiVersion = "1.8.3"
 
 # jetbrains compose
-composeJetbrainsVersion = "1.6.0"
+composeJetbrainsVersion = "1.8.2"
```

Comment:
These had to be upgraded to versions that support Kotlin 2.0 and its newly-removed native targets.